### ISSUE TITLE
Nested creates (and possibly updates) 

### ISSUE THREAD

**Issue by mikermcneil** [balderdashy/waterline#412]
@particlebanana stumbled onto this one:

Nested creates (and possibly updates) on `collection` associations don't work when empty objects are used.

For example:

```js
User.create({
  "name": "roger",
  "lessons": [{}, {}]
})
```
yields:
```
[ { type: 'insert',\n    collection: 'lesson_users__user_lessons',\n    criteria: { lesson_users: {}, user_lessons: 6 },\n    values: { lesson_users: {}, user_lessons: 6 },\n    err: 'Associated Record For lesson with id = [object Object] No Longer Exists' },\n  { type: 'insert',\n    collection: 'lesson_users__user_lessons',\n    criteria: { lesson_users: {}, user_lessons: 6 },\n    values: { lesson_users: {}, user_lessons: 6 },\n    err: 'Associated Record For lesson with id = [object Object] No Longer Exists' } ]
```

But it works fine if you pass the following to Waterline:
```json
{
  "name": "roger",
  "lessons": [{"name": "history"}, {"name": "algemath"}]
}
```

**Comment by devinivy** [comment]
Any word on this?  I wonder what happens when creating an empty object in general.

**Comment by dmarcelino** [comment]
I think we should start by creating a test in [create.nested.js](https://github.com/balderdashy/waterline-adapter-tests/blob/master/interfaces/associations/manyToMany/create.nested.js). That would give us an idea if this is still an issue and if waterline core or some adapters need fixing.

**Comment by sailsbot** [comment]
Thanks for posting, @mikermcneil.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by mrded** [comment]
In Waterline 0.13 there no nested creates or updates. Is there a way to still do that?
